docs/guides/admin/docs/upgrade.md

```diff
@@ -82,11 +83,61 @@ change `maintenance` back to `false` to put the scheduler back into its normal m
 You should avoid running Opencast 7 without migrating the scheduled events first. Otherwise, your capture agents may
 fetch an empty calendar.
 
+
+Update Player Links
+-------------------
+
+> This step is optional
+
+Opencast 7 comes with the capability of dynamically switching the configured player without requiring the republication
```

Comment:
@lkiesow can you clarify that this the publish link from the Admin UI. For example, something like: "...dynamically switching the configured player that is referenced from the Admin UI without requiring...."

Reply:
But it's not just the admin interface. It's basically all places: admin interface, external API, LTI tools, media module, search endpoint (I think that's all)